docs/welcome.mdx

```diff
@@ -5,31 +5,31 @@ sidebar_label: Introduction
 ---
 
 Ory is a software infrastructure provider building a global zero-trust network for humans, robots, devices, and software services.
-Ory develops open-source software on [GitHub](https://github.com/ory) and publishes open standards such as the
+Ory develops open source software on [GitHub](https://github.com/ory) and publishes open standards such as the
```

Comment:
`open-source` here is an adjective - why did you remove the hyphen?

Reply:
My bad, it didnt look right to me 😓 
I will add it again for the adjectives!
We should document it or link to the microsoft styleguide somewhere:
```
Hyphenate open-source as an adjective preceding a noun, as in open-source software. Don't use open-sourced as an adjective.

Don't hyphenate in other instances, such as : "Open source is a development model in which ...."

Don't use OSS as an abbreviation for open-source software.
```